examples/fluids/index.md

```diff
@@ -634,19 +634,26 @@ The `STGRand.dat` file is the table of the random number set, $\{\bm{\sigma}^n,
 The following table is presented to help clarify the dimensionality of the
 numerous terms in the STG formulation.
 
-| Math            | Label  | $f(\bm{x})$? | $f(n)$? |
-|-----------------|--------|--------------|---------|
-| $ \{\bm{\sigma}^n, \bm{d}^n, \phi^n\}_{n=1}^N$        | RN Set | No           | Yes     |
-| $\bm{\overline{u}}$ | ubar | Yes | No |
-| $U_0$           | U0     | No           | No      |
-| $l_t$           | l_t    | Yes          | No   |
-| $\varepsilon$   | eps    | Yes          | No   |
-| $\bm{R}$        | R_ij   | Yes          | No      |
-| $\bm{C}$        | C_ij   | Yes          | No      |
-| $q^n$           | q^n    | Yes           | Yes     |
-| $\{\kappa^n\}_{n=1}^N$ | k^n  | No           | Yes      |
-| $h_i$           | h_i    | Yes          | No   |
-| $d_w$           | d_w    | Yes          | No   |
+| Math                                           | Label    | $f(\bm{x})$?   | $f(n)$?   |
+| -----------------                              | -------- | -------------- | --------- |
+| $ \{\bm{\sigma}^n, \bm{d}^n, \phi^n\}_{n=1}^N$ | RN Set   | No             | Yes       |
+| $\bm{\overline{u}}$                            | ubar     | Yes            | No        |
+| $U_0$                                          | U0       | No             | No        |
+| $l_t$                                          | l_t      | Yes            | No        |
+| $\varepsilon$                                  | eps      | Yes            | No        |
+| $\bm{R}$                                       | R_ij     | Yes            | No        |
+| $\bm{C}$                                       | C_ij     | Yes            | No        |
+| $q^n$                                          | q^n      | Yes            | Yes       |
+| $\{\kappa^n\}_{n=1}^N$                         | k^n      | No             | Yes       |
+| $h_i$                                          | h_i      | Yes            | No        |
+| $d_w$                                          | d_w      | Yes            | No        |
+
+#### Internal Damping Layer (IDL)
+STG creates spurious acoustic waves during it's development to natural
+turbulent structures. To damp those out, we use an internal damping layer (IDL)
+to prevent them from growing. This implementation was inspired from
+{cite}`shurSTG2014`, but is implemented here as a ramped volumetric forcing
+term in the residual itself.
```

Comment:
Our formulation is based on Eq 8.32 of this chapter in the book John/Ken/I contributed to. https://doi.org/10.1016/B978-0-32-391144-3.00014-0
It has citations to prior papers, though I didn't read those. We should explain that the damping is toward `reference_pressure` in primitive variables (so no damping of velocity or temperature) linearized around the current state.

Reply:
Let me know if the current explanation make sense, then I'll go ahead and merge.